### PR TITLE
service pack 1

### PR DESCRIPTION
**Pull request by alex-nesterov**
hi!
i created "service pack 1" with next futures:

- buttons with stats on main page
- toggle task details
- optional naming for tasks
- buttons to pause and resume queues
- enabled `Code` mode at jsonEditor

**Demo**
bull-arena_service-pack-1

### CODE REVIEW

**Comment by alex-nesterov** [comment]
@ttacon its your turn now

**Comment by Subv** [comment]
Hi!
Are there plans to merge & release this any time soon? We would greatly benefit from this functionality (Thanks @alex-nesterov !)

**Comment by bvallelunga** [comment]
@alex-nesterov Would love to see if we can merge this soon! We would love this feature as well.

File: public/dashboard.js [hunk 1]
```diff
@@ -153,4 +155,36 @@ $(document).ready(() => {
...
+    }).done(() => {
+      alert('Queue successfully resumed!');
+    }).fail((jqXHR) => {
+      window.alert('Failed to resume queue, check console for error.');
+      console.error(jqXHR.responseText);
+    });
+  });
+
+  $('.js-toggle-collapse').on('click', function () {
+    $('.collapse').collapse('toggle');
```
Comment (skeggse) [inline public/dashboard.js:188]: For codebase maintainability, please us a `js-` prefixed class when interacting with elements from js. This helps folks refactor html/hbs files without needing to look at js source.

**Pull request closed without merging**